js/charts/heatmap.js

```diff
@@ -531,7 +531,9 @@ function heatmap() {
             .style('font-weight', fontWeightForDimension)
             .style('font-size', fontSizeForDimension)
             .style('text-anchor', 'end')
-            .attr('transform', 'translate(' + -offset + ',' + cellHeight / 1.75 + ')');
+            .attr('transform', 'translate(' + -offset + ',' + cellHeight / 1.75 + ')')
```

Comment:
create a generic function since this line is duplicate ...

(' + -offset + ',' + cellHeight / 1.75 + ')')